examples/ngsi_v2/e13_ngsi_v2_secure_fiware_headers.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
@sbanoeon, could you add a short description of what this example is about at the beginning? Just make sure the style aligns with the other examples we already have